process/design-review-best-practices.md

```diff
@@ -46,7 +46,7 @@ Be mindful of time and table or reschedule topics as needed.
 
 If possible, assign someone to take notes for you. This will allow you more headspace to engage in the conversation.
 
-For detailed tips on how to take part in a review, see our follow-up guide: best practices for giving and receiving feedback. 
+For detailed tips on how to take part in a review, see our follow-up guide: [best practices for giving and receiving feedback](https://github.com/labzero/guides/blob/master/process/giving-and-receiving-design-feedback.md). 
```

Comment:
I think this can be a relative link:

```suggestion
For detailed tips on how to take part in a review, see our follow-up guide: [best practices for giving and receiving feedback](giving-and-receiving-design-feedback.md).
```

The above appeared to work in "preview" mode.

Reply:
It looks like I'll be going through and updating all the links here.  
You can either use the commit suggestion as listed above, or just merge this and I'll update it later.